etl/steps/data/garden/hyde/2017/baseline.mapping.json

```diff
@@ -0,0 +1,238 @@
+  "Saint Lucia": "Saint Lucia",
+  "Samoa": "Samoa",
+  "San Marino": "San Marino",
+  "Saudi Arabia": "Saudi Arabia",
+  "Senegal": "Senegal",
+  "Serbia": "Serbia",
+  "Serbia and Montenegro": "Serbia and Montenegro",
+  "Seychelles": "Seychelles",
+  "Sierra Leone": "Sierra Leone",
+  "Singapore": "Singapore",
+  "Slovakia": "Slovakia",
+  "Slovenia": "Slovenia",
+  "Solomon Islands": "Solomon Islands",
+  "Somalia": "Somalia",
+  "South Africa": "South Africa",
+  "Spain": "Spain",
+  "Sri Lanka": "Sri Lanka",
+  "Sudan": "Sudan",
+  "Suriname": "Suriname",
+  "Swaziland": "Eswatini",
+  "Sweden": "Sweden",
+  "Switzerland": "Switzerland",
+  "Taiwan": "Taiwan",
+  "Tajikistan": "Tajikistan",
+  "Thailand": "Thailand",
+  "Togo": "Togo",
+  "Tokelau": "Tokelau",
+  "Tonga": "Tonga",
+  "Tunisia": "Tunisia",
+  "Turkey": "Turkey",
+  "Turkmenistan": "Turkmenistan",
+  "Tuvalu": "Tuvalu",
+  "Uganda": "Uganda",
+  "Ukraine": "Ukraine",
+  "United Kingdom": "United Kingdom",
+  "United States": "United States",
+  "Uruguay": "Uruguay",
+  "Uzbekistan": "Uzbekistan",
+  "Vanuatu": "Vanuatu",
+  "Venezuela": "Venezuela",
+  "Viet Nam": "Vietnam",
+  "Western Sahara": "Western Sahara",
+  "Yemen": "Yemen",
+  "Zambia": "Zambia",
+  "Zimbabwe": "Zimbabwe",
+  "Antigua & Barb.": "Antigua and Barbuda",
+  "Br.Virgin Is.": "British Virgin Islands",
+  "Bsnia.&Hrzgvna.": "Bosnia and Herzegovina",
+  "Central African Rep.": "Central African Republic",
+  "Christmas Isl.": "Christmas Island",
+  "Cocos (Keeling) Isl.": "Cocos Islands",
+  "Czech Rep.": "Czechia",
+  "Dem.Rep.,Korea": "North Korea",
+  "Dominican Rep.": "Dominican Republic",
+  "East Timor": "Timor",
+  "Eq. Guinea": "Equatorial Guinea",
+  "Falkland Is.": "Falkland Islands",
+  "Guadaloupe": "Guadeloupe",
+  "Heard and McDonald Isl.": "Heard Island and McDonald Islands",
+  "Holy See": "Vatican",
+  "Iran, Isl.Rep.of": "Iran",
+  "Lao": "Laos",
+  "Libyan A.J.": "Libya",
+  "Micronesia (Fed. States of)": "Micronesia (country)",
+  "Neth.Antilles": "Netherlands Antilles",
+  "Norfolk Isl.": "Norfolk Island",
+  "Northern Mariana": "Northern Mariana Islands",
+  "Occupied Palestinian Terr.": "Palestine",
+  "Rep. of Korea": "South Korea",
+  "Rep. of Moldova": "Moldova",
+  "Russian Fed.": "Russia",
+  "Sao Tome & Pr.": "Sao Tome and Principe",
+  "St. Helena": "Saint Helena",
+  "St.Kitts & Nevis": "Saint Kitts and Nevis",
+  "St.Pierre & Miq.": "Saint Pierre and Miquelon",
+  "St.Vin. & Gren.": "Saint Vincent and the Grenadines",
+  "Svalbard & Jan Mayen Isl.": "Svalbard and Jan Mayen",
+  "Syrian Arab Rep.": "Syria",
+  "TFYR Macedonia": "North Macedonia",
+  "Trin. & Tobago": "Trinidad and Tobago",
+  "Turks & Caicos": "Turks and Caicos Islands",
+  "U.Arab Emirates": "United Arab Emirates",
+  "U.R. Tanzania": "Tanzania",
+  "U.S. Virgin Is.": "United States Virgin Islands",
+  "Wallis & Futuna": "Wallis and Futuna",
+  "Zaire/ Congo , Dem. Rep.": "Democratic Republic of Congo"
```

Comment:
Everything after Zimbabwe in this file is manually mapped and should get review.

Reply:
Looks good (despite my poor geographic knowledge).